FirebaseRemoteConfig/Tests/HermeticAPI/APITests.swift

```diff
@@ -0,0 +1,177 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseCore
+@testable import FirebaseRemoteConfig
+
+import XCTest
+
+class APITests: XCTestCase {
+  var app: FirebaseApp!
+  var config: RemoteConfig!
+
+  override class func setUp() {
+    FirebaseApp.configure()
+  }
+
+  override func setUp() {
+    super.setUp()
+    app = FirebaseApp.app()
+    config = RemoteConfig.remoteConfig(app: app!)
+    let settings = RemoteConfigSettings()
+    settings.minimumFetchInterval = 0
+    config.configSettings = settings
+
+    FakeFetch.config = ["Key1": "Value1"]
+
+    // Uncomment for verbose debug logging.
+    FirebaseConfiguration.shared.setLoggerLevel(FirebaseLoggerLevel.debug)
+  }
+
+  override func tearDown() {
+    app = nil
+    config = nil
+    FakeFetch.config = nil
+    super.tearDown()
+  }
+
+  func testFetchThenActivate() {
```

Comment:
The first three tests are duplicated from the swift_api tests except now running with the fake instead of a real backend.